src/main/java/me/totalfreedom/totalfreedommod/EntityWiper.java

```diff
@@ -51,14 +51,17 @@ protected void onStop()
 
     // Methods for wiping
 
-    public int wipe()
+    public int wipe(boolean isScheduled)
     {
         int removed = 0;
         for (World world : Bukkit.getWorlds())
         {
             for (Entity entity : world.getEntities())
             {
-                if (!BLACKLIST.contains(entity.getType()) && !Groups.MOB_TYPES.contains(entity.getType()))
+                EntityType type = entity.getType();
+                if (!BLACKLIST.contains(type) && !Groups.MOB_TYPES.contains(type)
+                        // Don't kill bees on scheduled wipes (#171) #SaveTheBees
+                        && !(isScheduled && type == EntityType.BEE))
```

Comment:
`EntityType.BEE` is already in `MOB_TYPES` group...

Reply:
wait so where's the problem

Reply:
oh i see